js/homework.js

```diff
@@ -0,0 +1,94 @@
+// part 1
+var box = document.querySelector('.jumbotron');
+var donateButton = document.querySelector('.buttons .btn-primary');
+var volunteerButton = document.querySelector('.buttons .btn-secondary');
+
+//blue button
+var blueButton = document.querySelector('#blueBtn');
+blueButton.addEventListener('click', colorBlue);
+//orange button
+var orangeButton = document.querySelector('#orangeBtn');
+orangeButton.addEventListener('click', colorOrange);
+// green button
+var greenButton = document.querySelector('#greenBtn');
+greenButton.addEventListener('click', colorGreen);
+
+//blue button function
+function colorBlue() {
+    box.style.backgroundColor = '#588fbd';
+    donateButton.style.backgroundColor = '#ffa500';
+    volunteerButton.style.backgroundColor = 'black';
+    volunteerButton.style.color = 'white';
+}
+
+//orange button function
+function colorOrange() {
+    box.style.backgroundColor = '#f0ad4e';
+    donateButton.style.backgroundColor = '#5751fd';
+    volunteerButton.style.backgroundColor = '#31b0d5';
+    volunteerButton.style.color = 'white';
+}
+
+//green button function
+function colorGreen() {
+    box.style.backgroundColor = '#87ca8a';
+    donateButton.style.backgroundColor = 'black';
+    volunteerButton.style.backgroundColor = '#8c9c08';
+}
+
+//part 2    
+//input Text
+var textEmail = document.querySelector('#exampleInputEmail1');
+var textName = document.querySelector('#example-text-input');
+var textDescib = document.querySelector('#exampleTextarea');
+//submit button
+var sbumitButton = document.querySelector('form .btn-primary');
+sbumitButton.addEventListener('click', submitForm);
+
+//submit function
+function submitForm(event) {
+    event.preventDefault();
+    checkEmail();
```

Comment:
Thank you for your feedback.
I solve the problem by assign the validate functions results to three variables.Then check the variables to avoid run the functions twice 
thank you again.